### ISSUE TITLE
On Sierra, installing current MailTags keeps UM from working.

### ISSUE THREAD

**Issue by uberchet** [UniversalMailer/UniversalMailer#105]
I sure would like to use both things, but I can't because, with MT installed, UM doesn't work.

The only other plugin I use is Mail Act-On.

Thoughts?

**Comment by UniversalMailer** [comment]
I'm not sure on the internals of Mail Act-On but it might be that both plugins are using one or more identical APIs from Mail: because of the internal plugin's architecture in Mail.app only one plugin is assigned that resource. This means that the plugins will randomly be assigned the resource when you launch Mail.app, whoever wins the race condition wins. In this case I think the Mail Act-On plugin always wins because of its alphabetical precedence...

In other words: you can only use one of them...
